### PR TITLE
Update: support disable included nodes in tree picker

### PR DESCRIPTION
**Pull request by pphminions**
Ability to disable included nodes. Should set `node.isSelectable` to false

screen shot 2016-07-15 at 4 04 23 pm

screen shot 2016-07-19 at 11 30 08 am

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

**Comment by sharmajyo** [comment]
LGTM

File: src/components/adslotUi/TreePickerNodeComponent.js [hunk 1]
```diff
@@ -52,7 +52,12 @@ const TreePickerNodeComponent = ({
       <GridRow>
         {selected ?
           <GridCell classSuffixes={['button']}>
-            <Button block bsSize="xsmall" className="btn-inverse" onClick={removeNodeBound} disabled={disabled}>
+            <Button
+              block bsSize="xsmall"
+              className="btn-inverse"
+              onClick={removeNodeBound}
+              disabled={disabled || node.isSelectable === false}
```
Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.js]: for `+` button lower, we don't display it at all if `... && (node.isSelectable !== false)` do we want the `-` to behave differently?

Comment (pphminions) [inline src/components/adslotUi/TreePickerNodeComponent.js]: This one is already handling for the tree-picker, and for this design, we want the `-` button to be disabled.

Comment (omgaz) [inline src/components/adslotUi/TreePickerNodeComponent.js]: I assume all is ok, but just clarify with Pedros that this makes the sides inconsistent.

**Comment by omgaz** [comment]
LGTM

**Comment by pphminions** [comment]
@omgaz Please merge if the code is okay

**Comment by pphminions** [comment]
@omgaz, @OndrejCholeva - disabled the `+` as well when `isSelectable` false. This will apply to tree picker as well.

**Comment by omgaz** [comment]
LGTM when green

**Comment by OndrejCholeva** [comment]
LGTM, merging :rocket:

**Pull request merged**